src/plugin/cesium/cesiumrenderer.js

```diff
@@ -653,3 +653,19 @@ plugin.cesium.CesiumRenderer.prototype.flyToFeatures = function(features) {
     os.commandStack.addCommand(cmd);
   }
 };
+
+
+/**
+ * @inheritDoc
+ */
+plugin.cesium.CesiumRenderer.prototype.getMaxFeatureCount = function() {
+  return /** @type {number} */ (os.settings.get('maxFeatures.cesium', 150000));
```

Comment:
This could be implemented in `AbstractWebGLRenderer` without a breaking change by adding a class property for the settings key. Defaulting the key to `maxFeatures.3d` would be backward compatible with existing settings, and renderers would only need to change the property if they wanted their max feature count persisted separately from the default Cesium renderer.

Reply:
Removed the subclass functions and switch to using a key on the Abstract.